Libraries/LibWeb/CSS/StyleValues/GridTemplateAreaStyleValue.cpp

```diff
@@ -23,13 +23,15 @@ String GridTemplateAreaStyleValue::to_string() const
 
     StringBuilder builder;
     for (size_t y = 0; y < m_grid_template_area.size(); ++y) {
+        builder.append('"');
```

Comment:
Outputting quotes isn't guaranteed to work - escaped characters lose their escape during tokenizing, so they need to be added back in. `serialize_a_string()` in `CSS/Serialize.h` handles that.